app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px;
+  font-family: verdana, arial, helvetica, sans-serif;
```

Comment:
Maybe there's a way to define repeated definitions in a SASS variable?

Reply:
Actually, this file came in with the rails scaffold generation. We don't need it at all. I'll remove.